Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -181,133 +175,113 @@ internal void HandleAnimMsg(int stateHash, float normalizedTime, NetworkReader r
                 m_Animator.Play(stateHash, 0, normalizedTime);
             }
 
-            ReadParameters(reader, false);
+            ReadParameters(dirtyBits, reader);
         }
 
-        internal void HandleAnimParamsMsg(NetworkReader reader)
+        internal void HandleAnimParamsMsg(uint dirtyBits, NetworkReader reader)
         {
-            if (hasAuthority)
-                return;
+            if (hasAuthority) return;
 
-            ReadParameters(reader, true);
+            ReadParameters(dirtyBits, reader);
         }
 
         internal void HandleAnimTriggerMsg(int hash)
         {
             m_Animator.SetTrigger(hash);
         }
 
-        bool WriteParameters(NetworkWriter writer, bool autoSend)
+        uint GetDirtyBits()
```

Comment:
Can you make the DirtyBits an ulong instead?   that way we support up to 64 parameters instead of 32.   Note they are varinted in the [command], so it does not increase the bandwidth.

Reply:
Yeah sure, I used a uint because the parameter mask was already uint anyways. Would be a harmless change though, so why not!